### PR TITLE
SOF-1639 - add event for auto next started

### PR DESCRIPTION
**Pull request by vivanov90**

### CODE REVIEW

**Review by LindvedKrvang — APPROVED** [review]
Looks good, only minor comments :)

**Review by RasmusAlbrektsen — APPROVED** [review]

File: src/business-logic/services/rundown-timeline-service.ts [hunk 1]
```diff
@@ -142,33 +140,31 @@ export class RundownTimelineService implements RundownService {
...
   }
 
   private getEndStateForActivePart(rundown: Rundown): PartEndState {
-    return this.blueprint.getEndStateForPart(
-      rundown.getActivePart(),
-      rundown.getPreviousPart(),
-      Date.now(),
-      undefined
-    )
+    return this.blueprint.getEndStateForPart(rundown.getActivePart(), rundown.getPreviousPart(), Date.now(), undefined)
```
[the remark this answers is not in the document]
Comment (vivanov90) [reply src/business-logic/services/rundown-timeline-service.ts]: Done

File: src/presentation/services/rundown-event-service.ts [hunk 2]
```diff
@@ -145,4 +151,9 @@ export class RundownEventService implements RundownEventEmitter, RundownEventObs
   public subscribeToRundownEvents(onRundownEventCallback: (rundownEvent: RundownEvent) => void): void {
     this.callbacks.push(onRundownEventCallback)
   }
+
+  public emitAutoNextStarted(rundownId: string): void {
+    const event: AutoNextStartedEvent = this.rundownEventBuilder.buildAutoNextStartedEvent(rundownId)
+    this.emitRundownEvent(event)
+  }
```
[the remark this answers is not in the document]
Comment (vivanov90) [reply src/presentation/services/rundown-event-service.ts:154]: Done

File: src/business-logic/services/rundown-timeline-service.ts [hunk 3]
```diff
@@ -269,9 +265,7 @@ export class RundownTimelineService implements RundownService {
 
     await this.buildAndPersistTimeline(rundown)
 
-    const segmentId: string = pieceToBeReplaced.getPartId() === rundown.getActivePart().id
-      ? rundown.getActiveSegment().id
-      : rundown.getNextSegment().id
+    const segmentId: string = pieceToBeReplaced.getPartId() === rundown.getActivePart().id ? rundown.getActiveSegment().id : rundown.getNextSegment().id
```
[the remark this answers is not in the document]
Comment (vivanov90) [reply src/business-logic/services/rundown-timeline-service.ts]: Done

**Pull request merged**